### ISSUE TITLE
Is Notion API stable enough to use for blogs?

### ISSUE THREAD

**Issue by seong7** [transitive-bullshit/nextjs-notion-starter-kit#163]
I just found an issue which seems to be of Notion API.
Your blog (https://transitivebullsh.it/) also is not showing any contents like below. 
<img src="https://user-images.githubusercontent.com/52827441/136129123-bf99ce6a-06d5-4eac-a5b0-b30f3faddf26.png" width="400"/>
It is happening to my blog which is thankfully built based on this repo as well.

Although I've seen such issue only once this time, but got curious if Notion API is stable enough to use for my blog.
I actually didn't even know it yet is only a beta.

I just want to hear opinions of yours and anyone's who loves this project and build their blogs using it.

**Comment by pzjzeason** [comment]
NotionAPI collectionQuery error Response code 400 (Bad Request), so the collection block doesn't work. Need help.

**Comment by pzjzeason** [comment]
check out here: https://github.com/NotionX/react-notion-x/issues/139#issue-1017271509

**Comment by normdoow** [comment]
> check out here: [NotionX/react-notion-x#139 (comment)](https://github.com/NotionX/react-notion-x/issues/139#issue-1017271509)

Yep, this is what is causing this problem. Just happened today. only the second breaking change that Notion has made like this in probably the last 9 months. So its pretty stable. I am working on making a fix for this  right now.

**Comment by youngle316** [comment]
`npm run deps:update`

**Comment by XahidEx** [comment]
> > check out here: [NotionX/react-notion-x#139 (comment)](https://github.com/NotionX/react-notion-x/issues/139#issue-1017271509)
> 
> Yep, this is what is causing this problem. Just happened today. only the second breaking change that Notion has made like this in probably the last 9 months. So its pretty stable. I am working on making a fix for this right now.

Have you fixed it?

**Comment by XahidEx** [comment]
> @XahidEx yes. but I don't think someone has updated notion-starter-kit to take the updates. But react-notion-x has been fixed and the starter kit depends on it. you should be able to update it yourself by going to the packa.json and updating the react-notion-x and notion-client versions. ![image](https://user-images.githubusercontent.com/21371266/136709225-fbd725d4-5e1a-4624-9b92-41c2eade5a92.png)

Still getting errors. 
![Screenshot_20211011-045849_Brave](https://user-images.githubusercontent.com/88249321/136715648-d5011418-7b48-4bdf-b651-beb5be4744a0.jpg)
Have a look

**Comment by XahidEx** [comment]
![Screenshot_20211011-050451_Brave](https://user-images.githubusercontent.com/88249321/136715764-750159cb-76c6-4a29-93cc-6a594f9a4cd2.jpg)

**Comment by XahidEx** [comment]
![Screenshot_20211011-050451_Brave](https://user-images.githubusercontent.com/88249321/136715784-49c4a7fa-a669-4e0b-8334-344b9c0d829d.jpg)

**Comment by XahidEx** [comment]
> @XahidEx yes. but I don't think someone has updated notion-starter-kit to take the updates. But react-notion-x has been fixed, and the starter kit depends on it. you should be able to update it yourself by going to the packa.json and updating the react-notion-x and notion-client versions. ![image](https://user-images.githubusercontent.com/21371266/136709225-fbd725d4-5e1a-4624-9b92-41c2eade5a92.png)

Fixed! But got another issue. Why my blog posts on the left side? 
![Screenshot_20211011-051800_Brave](https://user-images.githubusercontent.com/88249321/136716113-99cc70d9-c96a-4677-8640-b003707cb77f.jpg)

**Comment by transitive-bullshit** [comment]
This should now be fixed in `main`.

If you have separate styling issues, please open a new issue.
